staff-registration-process.php

```diff
@@ -0,0 +1,38 @@
+<?php 
+require 'connection.php';
+
+$fName = $_POST["firstName"];
+$lName = $_POST["lastName"];
+$email = $_POST["email"];
+$password = $_POST["password"];
+$contact = $_POST["contact"];
+$role = $_POST["role"];
+
+if (
+    empty($fName) || empty($lName) || empty($email) ||
+    empty($password) || empty($contact)
+) {
+    echo "Please fill the required areas...!";
+}else if($role == "Select here"){
```

Comment:
This "else-if validation" is useless. Because you don't put any value to "value" attribute in "[staff-registration.php](https://github.com/DilushaLakshan/RoamRadienceTravelers/pull/2/files#diff-5fdeb863cc7c1530e630f45f06f3146e8b3652b925157be704bd7d30f9fc2d36)" file. Check that file

Reply:
OK.